python_src/src/lamp_py/performance_manager/l1_rt_trips.py

```diff
@@ -183,7 +183,7 @@ def load_new_trip_data(db_manager: DatabaseManager) -> None:
             VehicleTrips.trip_id == distinct_update_query.c.trip_id,
         )
     )
-    db_manager.execute(trip_update_query)
+    db_manager.execute(trip_update_query, disable_trip_tigger=True)
```

Comment:
is the idea that we'll run this trigger _once_ per event loop rather than on each thing that touches the `VehicleTrips`?

Reply:
Yes, we expect the trigger to get run as part of the `update_trip_stop_counts` function, which we have commented as such. But specifically disabling for all other quries that are updating `VehcileTrips` should be a major speed up. 

Using my staging DB extract for testing, the `update_static_version_key` call drops from ~100 seconds to 38ms.

Reply:
It's a little hacky, but seems reasonable for now??